features.txt

```diff
@@ -0,0 +1,8 @@
+RRR Features:
+    Movement:
+        8 directional
+    Shooting:
+        to mouse location

```

Comment:
Needs to be more detailed -- Ex: What design patterns will be used for the shooting? For instance, a base weapon class that all weapons can inherit from? Be more specific otherwise you're off the team, bitch.

Reply:
> Needs to be more detailed -- Ex: What design patterns will be used for the shooting? For instance, a base weapon class that all weapons can inherit from? Be more specific otherwise you're off the team, bitch.

wtf dude